docs/running-on-kubernetes.md

```diff
@@ -1018,6 +1047,13 @@ See the [configuration page](configuration.html) for information on Spark config
     Request timeout in milliseconds for the kubernetes client in driver to use when requesting executors.
   </td>
 </tr>
+<tr>  
+  <td><code>spark.kubernetes.submitGracePeriod</code></td>
```

Comment:
The name is a little weird: it mentions "submit" but it's used when killing apps.

Reply:
Its because submit is used for the ops like `kill`. Maybe `DeleteGracePeriodForAppKill`?

Reply:
`killGracePeriod` seems good enough for me.